### PR TITLE
PR template

### PR DESCRIPTION
**Pull request by SHADOWELITE7**
I was in quite of a hurry to go somewhere. You can add more stuff if you like if you don't like the way it is currently

### CODE REVIEW

**Comment by SHADOWELITE7** [comment]
hmm I guess I should have read the [help article](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/)

**Comment by willeastcott** [comment]
Can I get some context on this? What is this exactly?

**Comment by SHADOWELITE7** [comment]
A PR or pull request template is for others to show the terms to make a PR merge able
Like you know one of guy called Frank's PR had something that would break the playcanvas engine.
I was thinking of adding the project template but I didn't have much time because I had to go to the gym. 
You can make your own edits if you like.

**Comment by SHADOWELITE7** [comment]
https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Pull Request template
+
+Here you can sen your fork of the playcanvas/engine to add your own features and updates
```
Comment (SHADOWELITE7) [inline PULL_REQUEST_TEMPLATE.md]: I don't think we need this part of the PR template do we

**Comment by SHADOWELITE7** [comment]
```Here you can sen your fork of the playcanvas/engine to add your own features and updates```
do you think this line should be deleted

**Comment by willeastcott** [comment]
I don't really want to impose a PR template on contributors at the moment. However, thanks for the suggestion. As a result of this, I've improved various other things in the repo:

* I've added an issue template
* I've added a code of conduct
* I've renamed CONTRIB.md to CONTRIBUTING.md, which is the GitHub official naming convention

Closing this now...

**Pull request closed without merging**